blueocean-dashboard/src/main/js/components/MultiBranch.jsx

```diff
@@ -51,7 +51,7 @@ EmptyState.propTypes = {
 
 export class MultiBranch extends Component {
     componentWillMount() {
-        if (this.context.config && this.context.params) {
+        if (this.context.pipeline && this.context.params && !pipelineBranchesUnsupported(this.context.pipeline)) {
```

Comment:
@kzantow it works fine, why do you thing it would not?

furthe in PipelinePage the parent we have 

```
        if (pipeline && pipeline.$failed) {
            return <NotFound />;
        }
```

which prevents that from failing